src/plugins/rpd/rpdplugin.cpp

```diff
@@ -0,0 +1,319 @@
+#include "rpdplugin.h"
+
+#include "maptovariantconverter.h"
+#include "objectgroup.h"
+#include "savefile.h"
+#include "varianttomapconverter.h"
+
+#include "qjsonparser/json.h"
+
+#include <QFile>
+#include <QFileInfo>
+#include <QJsonArray>
+#include <QJsonDocument>
+#include <QJsonObject>
+#include <QTextStream>
+#include <QtMath>
+
+namespace Rpd {
+
+void RpdPlugin::initialize()
+{
+    addObject(new RpdMapFormat(RpdMapFormat::Rpd, this));
+    addObject(new RpdTilesetFormat(this));
+}
+
+RpdMapFormat::RpdMapFormat(SubFormat subFormat, QObject *parent)
+    : Tiled::WritableMapFormat(parent)
+    , mSubFormat(subFormat)
+{}
+
+QString RpdMapFormat::shortName() const
+{
+    return "RPD";
```

Comment:
All other formats use lower-case short names, so I'd suggest to do the same here.
```suggestion
    return "rpd";
```